cmd/PolicyGenerator/main.go

```diff
@@ -2,20 +2,48 @@ package main
 
 import (
 	"bytes"
+	_ "embed"
 	"fmt"
 	"os"
+	runtimeDebug "runtime/debug"
 
 	"github.com/spf13/pflag"
 	"open-cluster-management.io/policy-generator-plugin/internal"
 )
 
+//go:embed VERSION.txt
+var version []byte
```

Comment:
I really like this (and didn't know `git describe` existed!), except that `stolostron` uses branches instead of tags, so `git describe --dirty` doesn't return anything. But there could be a function in the Makefile to fill it in using the branch/commit if it's empty...